specification/app/data-plane/readme.md

```diff
@@ -0,0 +1,56 @@
+# Container Apps Data Plane Api
```

Comment:
1.  Let's move these data-plane specs under this folder  `/specification/app/data-plane/Microsoft.App/SessionPools` as root folder 

similar to [Storage service ](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/storage/data-plane) but different resource types rather different namespace like Storage have.

Another example closest (cognitiveservices](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/cognitiveservices/data-plane)

2. then we have two folders `preview` and `stable` 

3. Let's include current live API version spec as it is, for customers to compare the differences easily during our migration guide as new stable API version has breaking changes.

Reply:
Resolved